dso/model.py

```diff
@@ -118,8 +119,18 @@ class MalwareSummary:
     findings: list[clamav.client.ScanResult]
 
 
+@dataclasses.dataclass(frozen=True)
+class Codecheck:
+    high: int
+    medium: int
+    low: int
+    info: int
+    scanRisk: int
```

Comment:
this name seems odd. should this not rather be named sth. like `risk_rating` (or just `risk`)?